src/sqlitetablemodel.h

```diff
@@ -11,6 +11,11 @@ namespace sqlb { class ForeignKeyClause; }
 class SqliteTableModel : public QAbstractTableModel
 {
     Q_OBJECT
+
+#ifdef REGEX_UNIT_TEST
+    friend class TestRegex;
```

Comment:
Simple.
The method we are testing (removeStuffFromOtherStuff) is private. We can't access private functions from beyond, we can only call them within this class.
So what we are doing here is just declaring TestRegex as a friendly class that can access any private method/member. **But** only and only *if* TestRegex is a thing. 
If we are doing a simple SQLiteBrowser build - there are no tests at all, REGEX_UNIT_TEST isn't defined anywhere, so `friend class TestRegex` is skipped.
Now imagine we are running tests.
In that TestRegex.cpp file you will see line
```
#define REGEX_UNIT_TEST
```
This is a trigger for SqliteTableModel.h. It acknowledges that the header file was requested by TestRegex hence it wants to gain *full* access to all members of this particular class. So that when we are calling model.removeComments...() in TestRegex.cpp it doesn't complains that method is private and stuff.
Capish? :)